### DISCUSSION TITLE
Running the samples?

### DISCUSSION THREAD

**Discussion by mrochon** [microsoft/TypeChat#118] [Q&A] [answered]
I am trying to follow [instructions to run the samples](https://microsoft.github.io/TypeChat/docs/examples/) using codespace. However, *npm run build-all* command fails with error 'ts not found' for each example folder. What am I missing?

`> npm run build --workspaces

> typechat@0.0.10 build
> tsc -p src

sh: 1: tsc: not found
npm ERR! Lifecycle script `build` failed with error: 
npm ERR! Error: command failed 
npm ERR!   in workspace: typechat@0.0.10 
npm ERR!   at location: /workspaces/TypeChat 

> calendar@0.0.1 build
> tsc -p src

sh: 1: tsc: not found
npm ERR! Lifecycle script `build` failed with error: 
npm ERR! Error: command failed 
npm ERR!   in workspace: calendar@0.0.1 
npm ERR!   at location: /workspaces/TypeChat/examples/calendar 
`

Following some hints on SO I run 'npm install typescript -g' and then the errors changed to:
`
> typechat@0.0.10 build-all
> npm run build --workspaces

> typechat@0.0.10 build
> tsc -p src

error TS2688: Cannot find type definition file for 'node'.
  The file is in the program because:
    Entry point of type library 'node' specified in compilerOptions

  src/tsconfig.json:6:15
    6     "types": ["node"],
                    ~~~~~~
    File is entry point of type library specified here.

`

**Comment by DanielRosenwasser** [comment, marked as answer]
I know it's been a while, but it looks you didn't run the `npm install` command.
